### PR TITLE
#32: swagger documentation added. though i could not do testing becau…

### PR DESCRIPTION
**Pull request by abdulrahmanalbakkar**
…se there are comments on lines 13 and 18 in app.js that make errors when removed.

### CODE REVIEW

File: src/app.js
Comment (atakann) [inline src/app.js:1]: This is 2 space indentation in the current app.js, you should git pull first, all of these were fixed in latest PR

File: src/db.js
Comment (atakann) [inline src/db.js:1]: Again all these indentations were already fixed with the most recent PR

**Review by atakann — CHANGES_REQUESTED** [review]

File: src/app.js
Comment (abdulrahmanalbakkar) [reply src/app.js:1]: I already did a git pull but I think I didn't get the latest version of the repository as your PR has not yet been approved. I'll do another pull request after your pull request gets approved.

**Review by Younesnz — CHANGES_REQUESTED** [review]
please check the indentations. they should be 2. it will cause merge conflicts in future with other team members work.
also Where is swagger JSON? I couldn't find it!
I suggest you to pull again and in vscode settings and also in .prettierrc.json change TabWidth to 2. and you should add something to the files. the only changes that I see are in Tab Widths. nothing else...

**Pull request closed without merging**